Server/Scripts/configure.ps1

```diff
@@ -25,15 +26,15 @@ $env:OCTOPUS_INSTANCENAME=$OctopusInstanceName
 function Validate-Variables() {
     # If either a username or password has been set, the other is set to a default. We also set
     # default creds if no master key is supplied.
-    if ($octopusAdminPassword -ne $null -or $octopusAdminUserName -ne $null -or !$masterKeySupplied)
+    if ($credentialsSupplied -or !$masterKeySupplied)
     {
       if ($octopusAdminUsername -eq $null)
       {
-        $octopusAdminUsername = "admin"
+        $script:octopusAdminUsername = "admin"
       }
       if ($octopusAdminPassword -eq $null)
       {
-        $octopusAdminPassword = "Passw0rd123"
+        $script:octopusAdminPassword = "Passw0rd123"
```

Comment:
the `$script:` prefix is the key change. Previously the default wouldnt be applied

Reply:
Good old PowerShell scoping. There's often a more elegant way than explicit scoping like this - like maybe returning a hashtable out of this function to keep the scopes tight - but honestly, if it works, I'm fine with it